script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -437,22 +440,23 @@ This is no longer a battle.
 [sha:6e6b32cdf27c181af94a6b6ef0cb4f809c2d01f0]{
 -- Page 493, Offset 9909.
 -- 炎を無効化された時点で、ヴローヴは狩られる側の獲物になりさがった。
-As soon as his flames were nullified, Vlov became mere prey to be hunted.
+As soon as his flames were neutralized, Vlov became mere prey to be hunted.
 }
 [sha:b04965fb7bac6c4327806b4f7e4436adfacdd563]{
 -- Page 493, Offset 9910.
 -- アルクェイドの爪は的確に、間断なく、ヴローヴの肉体に致命傷を重ね続ける。
-Arcueid's claws are unrelenting, and continue to accurately inflict mortal wounds on his body.
+Arcueid's claws continue to unrelentingly inflict accurate, mortal wounds into Vlov's flesh.
```

Comment:
Mortal wounds applied to flesh reads slightly off to me, maybe because it's less personal / 'living' as a descriptor?

Reply:
It sorta feels wrong to me also but the image is so visceral that I'm fine with it.